irods/library/irods_unixfilesystem_resource

```diff
@@ -16,10 +16,10 @@ short_description: Create an iRODS Unix filesystem storage resource
 
 version added: "2.11.11"
 
-description: An ansible module for creating an iRODS Unix filesystem storage
-resource
+description: 
+    - An ansible module for creating an iRODS Unix filesystem storage resource
```

Comment:
This comment applies to all `descritption` fields in this section.

When YAML field holds a long string holding text that can be wrapped when rendered, use a `>` followed by a linefeed.  This tells YAML that the next set of indented lines are considered to be a single long string that can wrapped.  Linefeeds can be used to wrap the line to preserve readability in the editor.  For example,

```yaml
field_1: >
  Here is a paragraph that can be
  wrapped.

field_2: ...
```

See the first answer under https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines for more information.